internal/pkg/externalplugins/cherrypicker/cherrypicker_invite_test.go

```diff
@@ -0,0 +1,200 @@
+	}{
+		{
+			name:          "invite when he is a collabtor",
+			collaborators: []string{"wiseguy"},
+			commentUser:   "wiseguy",
+			isMember:      false,
+			wantError:     false,
+			wantHad:       true,
+		},
+		{
+			name:          "invite when he is a collabtor, but comment failed",
+			collaborators: []string{patternErrWhenCreateComment},
+			commentUser:   patternErrWhenCreateComment,
+			isMember:      false,
+			wantError:     true,
+			wantHad:       true,
+		},
+		{
+			name:        "invite when he is not a collabtor, but a org member",
+			commentUser: "wiseguy",
+			wantError:   false,
+			isMember:    true,
+			wantHad:     true,
+		},
+		{
+			name:        "invite when he is not a collabtor, but a org member, notify comment failed",
+			commentUser: patternErrWhenCreateComment,
+			wantError:   true,
+			isMember:    true,
+			wantHad:     true,
+		},
+		{
+			name:        "invite when he is not a collabtor, error when judge member",
+			commentUser: patternErrWhenJudgeMember,
+			wantError:   true,
+			isMember:    false,
+			wantHad:     false,
+		},
+		{
+			name:        "invite when he is not a member of our ORG",
+			isMember:    false,
+			commentUser: "wiseguy",
+			wantError:   false,
+			wantHad:     false,
+		},
+		{
+			name:        "invite when he is not a member of our ORG, error when comment",
+			isMember:    false,
+			commentUser: patternErrWhenCreateComment,
+			wantError:   true,
+			wantHad:     false,
+		},
+		{
+			name:          "invite error when judge",
+			collaborators: nil,
+			commentUser:   patternErrWhenIsCollaborator,
+			wantError:     true,
+			wantHad:       false,
+		},
+		{
+			name:          "invite error when add",
+			collaborators: nil,
+			isMember:      true,
+			commentUser:   patternErrWhenAddCollaborator,
+			wantError:     true,
+			wantHad:       false,
+		},
+		{
+			name:          "invite error when add, and comment failed",
+			collaborators: nil,
+			isMember:      true,
+			commentUser:   patternErrWhenAddCollaborator + patternErrWhenCreateComment,
+			wantError:     true,
+			wantHad:       false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ghc := &fghc{
+				pr: &github.PullRequest{
+					Base:      github.PullRequestBranch{Ref: "master"},
+					Number:    2,
+					Merged:    true,
+					Title:     "This is a fix for X",
+					Body:      body,
+					Assignees: []github.User{{Login: "user2"}},
+				},
+				isMember:      tt.isMember,
+				members:       tt.members,
+				patch:         patch,
+				collaborators: tt.collaborators,
+			}
+			ic := github.IssueCommentEvent{
+				Action: github.IssueCommentActionCreated,
+				Repo: github.Repo{
+					Owner:    github.User{Login: "foo"},
+					Name:     "bar",
+					FullName: "foo/bar",
+				},
+				Issue: github.Issue{Number: 2, State: "closed", PullRequest: &struct{}{}},
+				Comment: github.IssueComment{
+					User: github.User{Login: tt.commentUser},
+					Body: "/cherry-pick-invite",
```

Comment:
Better add test cases for command.

Reply:
this is for command comment: `/cherry-pick-invite`, could you provide more info?

Reply:
yes, you can add invalid commands as test cases.

case 1: `/cherry-pick-invite`
case 2: `/cherrypick-invite`
case 3: `/invaild-command`

Reply:
OK, those tests only test into the invite issue comment events.
I will add `case 3` tests for no matched issue comemnt events for outer handle method.